src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder results = new StringBuilder();
+        String importantSymbol = ",";
+        int operationTypeIndex = 0;
+        int amountIndex = 1;
+        int arraysTheoryMaxSize = 10;
+        int realSize = 0;
+        String[] type = new String[arraysTheoryMaxSize];
+        int[] amount = new int[arraysTheoryMaxSize];
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
```

Comment:
You are using `String.split()` which is good, but the way you're processing the array is not efficient. Consider refactoring the logic to avoid using a fixed size array and the `realSize` variable.